php/cubex/composer.json

```diff
@@ -4,7 +4,16 @@
   "type": "project",
   "prefer-stable": true,
   "require": {
-    "cubex/framework": "~4.16.0"
+    "cubex/framework": "~4.16.0",
+    "packaged/polyfill-ctype": "*",
+    "packaged/polyfill-mbstring": "*",
+    "packaged/polyfill-intl": "*",
+    "packaged/polyfill-php71": "*",
+    "packaged/polyfill-php72": "*",
+    "packaged/polyfill-php73": "*",
+    "packaged/polyfill-php74": "*",
+    "packaged/polyfill-php80": "*",
+    "packaged/polyfill-php81": "*"
```

Comment:
so many, there is not a metapackage depending on `php` **version**

Reply:
There is a single package, however with the way composer works, its not possible to replace packages with a single import :(

Reply:
ok